src/services/azure/credential.rs

```diff
@@ -0,0 +1,91 @@
+use anyhow::{anyhow, Result};
+use std::fmt::{Debug, Formatter};
+use std::mem;
+#[derive(Default)]
+pub struct Builder {
+    cred: Credential,
+}
+
+impl Builder {
+    pub fn access_name(&mut self, access_name: &str) -> &mut Self {
```

Comment:
Please keep consistent with azure's doc in which they use `AccountName`/`account_name`.

Reply:
For reference: the golang version of azblob uses `accountName` and `accountKey`.

https://github.com/Azure/azure-sdk-for-go/tree/main/sdk/storage/azblob